### DISCUSSION TITLE
Scrollbars on Textarea extending past rounded border

### DISCUSSION THREAD

**Discussion by HattieTavares** [saadeghi/daisyui#3066] [Q&A] [answered]
I didn't submit an issue because this isn't a bug so much as a design issue but I noticed on my textarea the scrollbar is extending past the corners on the rounded borders.  I'm using the textarea for something unusual, but it works perfectly otherwise. The problem is that MOST of the time the textarea is going to be scrollable and the scrollbars are just so ugly like this.

I purposely picked daisyui because I detest writing my own CSS and wanted to do as little of it as possible, other than basic layout stuff. I've tried to figure out how to adjust it on my own but so far nothing has worked, and to be honest I don't have the time to really spend on it due to my actual job. I ended up just disabling scrollbars instead but if the content is scrollable it really should show that. I just can't look at them.

Attaching two photos from the daisyui textarea page to show what I mean. I'm hoping someone who knows much more than I do about this will have a suggestion on how to fix?
![Screenshot (9)](https://github.com/saadeghi/daisyui/assets/82660400/05b42222-0a23-4c0d-a1ce-6f3d3358a1a0)
![Screenshot (10)](https://github.com/saadeghi/daisyui/assets/82660400/8814c477-cc17-4c89-a8e9-a17a65e746ea)

**Comment by saadeghi** [comment, marked as answer]
Yes, this is a challenge with Windows but there's not a standard CSS solution for that. Unfortunately Windows applies fixed scrollbars with their own style for all scrollable areas.  
And the challenge is, there's no way in CSS to add paddings to scrollbar. It only sticks to the edge and with Windows it's a sharp gray rectangle. A workaround would be applying style to a div wrapper instead which is not a optimal solution.  

**Another workaround**

daisyUI cupcake theme has more border radius for larger components like card and textarea and this big border radius makes the Windows scrollbar issue more obvious. Another workaround in this situation would be decreasing the border radius with a class name like `rounded-lg` or `rounded`.  


**daisyUI 4.12.2 improvement**

There's a new CSS rule "[scrollbar-color](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color)" which can change the colors of the existing scrollbar style. `scrollbar-color` is added to daisyUI in version `4.12.2` to make the scrollbar background transparent which makes it look better.  
You can update daisyUI to `4.12.2` to see.

Let me know if you have a question.
